README.md

```diff
@@ -1,71 +1,108 @@
 ```
+puppet-lint ~/modules/puppetlabs-mysql/manifests
+```
+
+### Fix issues automatically
+
+To instruct Lint to automatically fix any issues that it detects, use the `--fix` flag:
 
-You can also instruct puppet-lint to automatically fix issues it detects with the `--fix` flag.
 ```
-    puppet-lint --fix /modules
+puppet-lint --fix /modules
 ```
 
-Note that this can be combined with the `--only-checks` option to help avoid enormous patch sets.
+### Modify which checks to run
+
+Puppet Lint options allow you to modify which checks to run. You can disable any of the checks temporarily or permanently, or you can limit testing to specific checks.
+
+#### Run specific checks
+
+To run only specific checks, use the `--only-checks` option, with arguments specifying which checks to make: [TODO: can you specify more than one argument for this?]
+
 ```
-    puppet-lint --only-checks trailing_whitespace --fix modules/
+puppet-lint --only-checks trailing_whitespace
 ```
 
-You can also use [control comments](http://puppet-lint.com/controlcomments/) to disable checks
-within puppet code on a per line or per block basis using `#lint:ignore:<check_name>`:
+To avoid enormous patch sets when using the `--fix` flag, use the `--only-checks` option to limit which checks Puppet Lint makes:
+
 ```
-    class foo {
-      $bar = 'bar'
-      # This ignores the double_quoted_strings check over multiple lines
-      # lint:ignore:double_quoted_strings
-      $baz = "baz"
-      $gronk = "gronk"
-      # lint:endignore
-
-      # This ignores the 140chars check on a single line
-      $this_line_has_a_really_long_name_and_value_that_is_much_longer_than_the_style_guide_recommends = "I mean, a really, really long line like you can't believe" # lint:ignore:140chars
-    }
+puppet-lint --only-checks trailing_whitespace --fix modules/
 ```
 
-See puppet-lint --help for a full list of options.
+### Disable checks
 
-### Rake
+* To disable any of the checks when running the `puppet-lint` command, add a `--no-<check name>-check` flag to the command. For example, to skip the 140-character check, run:
 
-If you want to test your entire Puppet manifest directory, you can add
-`require 'puppet-lint/tasks/puppet-lint'` to your Rakefile and then run
+  ```
+  puppet-lint --no-140chars-check /path/to/my/manifest.pp
+  ```
 
-    rake lint
+* To always skip a given check, specify the flag in a `.puppet-lint.rc`. Puppet Lint checks for this file in both the current directory and your home directory. For example, to always skip the hard tab character check, create `~/.puppet-lint.rc` and include the line:
+
+  ```
+--no-hard_tabs-check
+  ```
 
-If you want to modify the default behaviour of the rake task, you can modify
-the PuppetLint configuration by defining the task yourself.
+* To disable checks within your Puppet code, use [control comments](http://puppet-lint.com/controlcomments/). You can disable checks on a per-line or per-block basis using `#lint:ignore:<check_name>`.
```

Comment:
Control comments could get their own section header. Also, you can disable all checks rather than specific ones.